packages/in_app_purchase/lib/src/in_app_purchase_connection/product_details.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// The class represents the information of a product.
+///
+/// A list of [ProductDetails] can be obtained from the [QueryProductDetailsResponse].
```

Comment:
nit, subjective: I think it would be more helpful to link to `Connection.queryProductDetails` here than the response data class specifically.